sph/include/sph/particles_data.hpp

```diff
@@ -33,6 +33,7 @@
 #include <cstdio>
 #include <iostream>
 #include <vector>
+#include
```

Comment:
leftover?

Reply:
it should have been <variant>, I'll fix that